packages/yoga/src/Result/native/Details.jsx

```diff
@@ -83,7 +83,7 @@ const ResultDetails = ({
                   <>
                     <Separator />
                     <StyledText variant={variant}>
-                      + {numberOfItemsLeft.toString().padStart(2, '0')}
+                      + {numberOfItemsLeft.toString()}
                       {limitLabel !== '' && ` ${limitLabel}`}
```

Comment:
Why do not change to:
```jsx
{!!limitLabel ? '' : `{limitLabel}`}
```
Or just

```jsx
{!!limitLabel && `{limitLabel}`}
```
Once we should not show nether `null`, empty or `undefined` as limit label.

Reply:
Thinking in cases that the `limitLabel` prop is set as `null` or `undefined` for instance (not just the default prop value).